src/data/code-samples/en/text-scale/jetpack-compose.md

```diff
@@ -0,0 +1,17 @@
+# Scale text - Jetpack Compose
+
+On Android, you can use [Scale-independent Pixels](https://developer.android.com/guide/topics/resources/more-resources.html#Dimension) to scale text. This unit ensures that the user's preferences are taken into account when determining the font size. We recommend to define the [`fontSize`](https://developer.android.com/reference/kotlin/androidx/compose/ui/text/TextStyle#fontSize()) property inside the [`Typography`](https://developer.android.com/reference/kotlin/androidx/compose/material3/Typography) object in your code to ensure consistency throughout your app.
```

Comment:
In Jetpack Compose

Reply:
Add https://developer.android.com/reference/kotlin/androidx/compose/ui/tooling/preview/PreviewFontScale

Reply:
```
@FontScalePreviews
@Composable
fun NonResizingTextPreview() {
  ExperimentsTheme {
    Box(modifier = Modifier.width(200.dp)) {
      Text(text = "This is resizing text with font scale ${LocalDensity.current.fontScale}")
    }
  }
}
```